lib/zendesk_apps_support/location.rb

```diff
@@ -0,0 +1,29 @@
+module ZendeskAppsSupport
+  module Location
+    # the numbers below match the enum values on the database, do not change them!
+    LOCATIONS_AVAILABLE = {
+      "zendesk" => {
+        "top_bar" => 1,
+        "nav_bar" => 2,
+        "ticket_sidebar" => 3,
+        "new_ticket_sidebar" => 4,
+        "user_sidebar" => 5,
+        "organization_sidebar" => 6,
+        "background" => 7
+      },
+      "zopim" => {
+        "chat_sidebar" => 8
+      }
+    }.freeze
+
+    class << self
+      def hosts
+        LOCATIONS_AVAILABLE.keys
+      end
+
+      def for(host)
+        LOCATIONS_AVAILABLE[host.to_s].keys
```

Comment:
Well, apart form that this class is it self defining one :) `ids_for` is okay, but maybe make host a named parameter so it's at least somewhat clear what arguments it takes? (or do we not have ruby 2.0 as a minimum requirement?)